app/uk/gov/hmrc/bindingtariffclassification/model/Event.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+import java.time.ZonedDateTime
+
+import uk.gov.hmrc.bindingtariffclassification.model.CaseStatus.CaseStatus
+import uk.gov.hmrc.bindingtariffclassification.model.EventType.EventType
+
+
+case class Event
+(
+  id: String,
+  details: Details,
+  userId: String,
+  caseReference: String,
+  timestamp: ZonedDateTime = ZonedDateTime.now()
+)
+
+sealed trait Details {
+  val `type`: EventType
```

Comment:
do we want this EventType under details or under Event?

Reply:
Same question for the `Case` object.
Do we want the `type` under `Application` or under `Case` ?